suite/tests/linux/clone.c

```diff
@@ -340,10 +340,32 @@ make_clone3_syscall(void *clone_args, ulong clone_args_size, void (*fcn)(void))
                    [clone_args_size] "m"(clone_args_size), [fcn] "m"(fcn)
```

Comment:
Please see https://dynamorio.org/page_code_reviews.html#sec_commit_messages.  Include a body in the PR description, plus a line for the issue reference: `Issue: #3699`.